### PR TITLE
@Injects to constructor dependency injection

### PR DESCRIPTION
**Pull request by TomasVotruba**
I don't know where we talked about it, but I got this in WIP for a long time.

It neededing [InjectToConstructorSniff](https://github.com/Symplify/CodingStandard/blob/master/src/Refactorer/NetteDI/InjectToConstructorInjectionSniff.php) + bit manual work.

I don't have time to check it, so please do so after review.

Thank you :+1:

### CODE REVIEW

**Comment by filipsuk** [comment]
Hey, I am testing it and I can't get it to work. Unfortunately I don't have enough knowledge about Nette DI to fix it. The problem is with Translator and AbstractBaseControl.

@TomasVotruba either you have to setup the project on your local machine or @jakubzaba has to look at it :)

One of the errors:
image

File: app/config/config.neon [hunk 1]
```diff
@@ -1,4 +1,5 @@
 includes:
+	- database.neon
```
Comment (TomasVotruba) [inline app/config/config.neon]: It's best practise to use `parameters` section to store credentials, not particular database extension.

These can be used on more places or dblayers (e.g. for Doctrine) with only 1 place to modify.

Comment (TomasVotruba) [inline app/config/config.neon]: It's best practise to use `parameters` section to store credentials, not particular database extension.

These can be used on more places or dblayers (e.g. for Doctrine) with only 1 place to modify.

**Comment by TomasVotruba** [comment]
It works!

image

**Comment by TomasVotruba** [comment]
Rebased on `master` after `testing` merge

File: README.md [hunk 2]
```diff
@@ -14,7 +14,11 @@
 1. Vytvořit novou MySQL databázi a importovat `events.sql.zip`
 2. Zkopírovat `app/config/templates/config.local.neon` do adresáře `app/config` a upravit konfiguraci
 3. Zkopírovat `phinx.yml.template` jako nový soubor `phinx.yml` a nastavit přístupy do databáze (viz [Phinx docs](http://docs.phinx.org/en/latest/configuration.html))
-4. Nainstalovat závislosti `composer install` a `bower install`
+4. Nainstalovat závislosti
+    ```
+    composer install
+    bower install
+    ```
```
Comment (TomasVotruba) [inline README.md:21]: I didn't notice this, so I've split it to 2 lines that are easy to copy, paster and run.

**Comment by filipsuk** [comment]
@TomasVotruba Does the admin interface work for you as well? try url `/admin/sources/`, seems still broken. 

It would be nice to have some tests, which would just to try to render some pages and look for exceptions.

**Review by jakubzaba — APPROVED** [review]

**Comment by TomasVotruba** [comment]
@filipsuk I will remember to check admin later as well.

Tests would be great, just too much work for me.

@tomasfejfar Had some easy to run tool ([sth like this](https://github.com/tomasfejfar/tests-primer)) to test url status code with PHPUnit, but I can't find now. Tom?

**Comment by TomasVotruba** [comment]
@jakubzaba Thanks for fixing and mergin :clap:

**Comment by tomasfejfar** [comment]
@TomasVotruba Nope, it's not done yet, sorry...

**Pull request merged**